pandas/core/groupby/generic.py

```diff
@@ -675,7 +682,7 @@ def nunique(self, dropna: bool = True) -> Series:
     @doc(Series.describe)
     def describe(self, **kwargs):
         result = self.apply(lambda x: x.describe(**kwargs))
-        if self.axis == 1:
+        if self.axis == 1 or not isinstance(result.index, MultiIndex):
```

Comment:
As @jreback commented, can check that it's not empty. I think the code would be more readable if we added it as a separate check without the transpose - I couldn't figure out why you'd want to take the transpose before realizing the frame is empty.

```
if result.empty:
    return result
elif self.axis == 1:
    return result.T
```